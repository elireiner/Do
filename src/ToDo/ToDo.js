import React, { Component } from 'react';

export default class Todo extends Component{
    render(){
        return(
           <>
           {this.props.children}
           </>
        )
    }
}