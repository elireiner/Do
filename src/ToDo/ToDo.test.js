import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ToDo from './ToDo';

describe('ToDo', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<ToDo/>, div);
        ReactDOM.unmountComponentAtNode(div);
    })
    it('renders UI as expected', () => {
        const tree = renderer.create(<ToDo/>).toJSON();
        expect(tree).toMatchSnapshot()
    })
})