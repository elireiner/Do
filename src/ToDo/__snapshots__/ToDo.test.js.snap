// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToDo renders UI as expected 1`] = `<li />`;
